### ISSUE TITLE
sourcery dropping parens that follow a splat, raising SyntaxError

### ISSUE THREAD

**Issue by wizpig64** [sourcery-ai/sourcery#132]
### Issue description or question

Hi hi,

Sourcery is suggesting a change in a conditional expression of mine. The DeMorgan logic is sound, but sourcery is rewriting the surrounding lines in a way that changes the behavior. There's a splat operator before the parenthesis holding the conditional, and sourcery just decides the parens are not needed and drops them, but leaves the splat, creating a SyntaxError.

```diff
 class SiteNavigationView(NavigationView)
     def navigation(self):
         return Navigation(
             NavLeaf(label="Lorem", viewname="view-lorem"),
-            *(
-                [NavLeaf(label="Ipsum", viewname="view-ipsum")]
-                if self.request.user.is_staff and not settings.CURRENT_BRANCH == "beta"
-                else []
-            ),
-            NavLeaf(label="Dolor", viewname="view-dolor"),
+            *[NavLeaf(label="Ipsum", viewname="view-ipsum")]
+            if self.request.user.is_staff and settings.CURRENT_BRANCH != "beta"
+            else [],
+            NavLeaf(label="Dolor", viewname="view-dolor")
         )

```

when i simplify the demorgan logic myself the suggestion to drop the parens does not stay.

### Sourcery Version

0.8.8

### Code editor or IDE name and version

Pycharm 2021.1.eap, python 3.6.latest

### OS name and version

Linux

**Comment by Hellebore** [comment]
Thanks for raising, will definitely fix for our next release.

**Comment by Hellebore** [comment]
Fixed in 0.9.1.
